content/api/local-installation.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Local installation
```

Comment:
I'm not sure how it will look once rendered, but this feels ambiguous -- _what_ is being installed.

I also wonder whether this page should be phrased more as "Getting editor support for the API" rather than "Installing the API". The difference is that the former is an intent, while the latter is a way to achieve it.